derive_builder_core/src/macro_options/darling_opts.rs

```diff
@@ -95,6 +95,25 @@ pub struct BuildFn {
     /// * If `validate` is specified, then this type must provide a conversion from the specified
     ///   function's error type.
     error: Option<Path>,
```

Comment:
I'm wondering if we should make this into a type that can either be a key-value pair with a path, or a list that accepts options about the generated error:


`error = "path"`


`error(validation = false)`

The advantage would be that it's then impossible to incorrectly disable the validation variant when using a custom error.

Reply:
I managed to implement it so it supports the two forms from your comment.